src/Common/CodeEditor/types.ts

```diff
@@ -48,8 +47,21 @@ export interface CodeEditorInterface {
     isKubernetes?: boolean
     cleanData?: boolean
     chartVersion?: any
+    minHeight?: number | string
 }
 
+type XORProps<P extends string, S extends string, T extends Record<P | S, any>> =
```

Comment:
would this be a useful util type that we can export from common-lib?

Reply:
Yes, can move to common types